apps/src/lab2/views/Lab2ShareDialogWrapper.tsx

```diff
@@ -48,11 +48,6 @@ const Lab2ShareDialogWrapper: React.FunctionComponent<
   const thumbnailUrl = null;
   // TODO: support abuse reporting.
   const exceedsAbuseThreshold = false;
-  // TODO: When we support publishing, we can use this logic to determine if we can publish
```

Comment:
We use [`submissionStatus` in `lab2`](https://github.com/code-dot-org/code-dot-org/blob/266082aefc57b9f799e2dd326eae3cddcdc86e28/apps/src/lab2/views/dialogs/ShareDialog.tsx#L159) to determine if a user can submit a project or not.